boards/cats_rev1Pro/Core/Src/sensors/ms5607.c

```diff
@@ -89,7 +89,9 @@ static void ms_read_bytes(MS5607 *dev, uint8_t command, uint8_t *pData, uint16_t
 }
 
 // Write command
-static void ms_write_command(MS5607 *dev, uint8_t command) { spi_transmit(dev->spi_bus, &command, 1); }
+static void ms_write_command(MS5607 *dev, uint8_t command) {
+	spi_transmit(dev->spi_bus, &command, 1);
+}
```

Comment:
It looks like your formatter is doing something wrong.. we increased the char limit to 120 per line I think.

Reply:
I did that manually because it did not look nice before xD